tests/data_files/print_c.pl

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env perl
+# Usage:
+#   print_c mbedtls_test_foo <file.pem
```

Comment:
This script seems to be working a lot of dark magic, could you please document a little bit more what it is supposed to do?